core/primitives/src/errors.rs

```diff
@@ -634,3 +645,52 @@ impl Display for ActionErrorKind {
         }
     }
 }
+
+#[derive(Eq, PartialEq, BorshSerialize, BorshDeserialize, Clone)]
+pub enum EpochError {
+    /// Error calculating threshold from given stakes for given number of seats.
+    /// Only should happened if calling code doesn't check for integer value of stake > number of seats.
+    ThresholdError(Balance, u64),
```

Comment:
Please change from the unnamed tuple to an inline struct. Otherwise it's hard to understand it